### DISCUSSION TITLE
How do I disable sorting for all columns except the ones with `canSort: true`?

### DISCUSSION THREAD

**Discussion by lmosele** [TanStack/table#2092] [General]
I'm struggling with what feels like a pretty common scenario. Doesnt seem like `defaultCanSort` works as described, unless I'm missing something.

**Comment by lmosele** [comment]
So for now I passed in a custom `sortable` value to `column` and used it in conjunction with the existing props:

```
 {...column.getHeaderProps(column.sortable && column.getSortByToggleProps())}
```

**Comment by tannerlinsley** [comment]
We may need to fix something in its implementation, but for now, you should be able to set the defaultColumn instead:

```js
useTable({
  defaultColumn: useMemo(() => ({
    canSort: true // or false
  }), [])
})
```
